x-pack/plugins/security_solution/public/management/components/endpoint_responder/command_render_components/get_processes_action.tsx

```diff
@@ -53,9 +53,10 @@ export const GetProcessesActionResult = memo<ActionRequestComponentProps>(
         ? {
             endpoint_ids: [endpointId],
             comment: command.args.args?.comment?.[0],
+            agent_type: command.commandDefinition?.meta?.agentType,
           }
         : undefined;
-    }, [endpointId, command.args.args?.comment]);
+    }, [endpointId, command.args.args?.comment, command.commandDefinition?.meta?.agentType]);
```

Comment:
Maybe you could destructure these to improve readability? 
`const { agentType } = command.commandDefinition?.meta || {};`